conf/ssmtp.conf

```diff
@@ -11,7 +11,7 @@ root=postmaster
 # The place where the mail goes. The actual machine name is required
 # no MX records are consulted. Commonly mailhosts are named mail.domain.com
 # The example will fit if you are in domain.com and your mailhub is so named.
-mailhub=172.17.0.1
+mailhub=DOCKER_GATEWAY
```

Comment:
please add a comment, that the run-apache.sh script will replace DOCKER_GATEWAY with docker gateway IP.